modules/node_modules/@ncigdc/containers/FileTable.js

```diff
@@ -139,6 +139,7 @@ export const FileTableQuery = {
                 total
```

Comment:
It's using the full array in at least one spot 
https://github.com/NCI-GDC/portal-ui/blob/a50bfe6d3069373d5da9a1165979abaada7318a5/modules/node_modules/%40ncigdc/containers/FileTr.js#L72